common/src/main/java/tachyon/Constants.java

```diff
@@ -62,6 +62,14 @@
 
   public static final int DEFAULT_HOST_RESOLUTION_TIMEOUT_MS = 5000;
 
+  // Service versions should be incremented every time a backwards incompatible change occurs.
+  public static final long BLOCK_MASTER_SERVICE_VERSION = 1;
+  public static final long BLOCK_WORKER_SERVICE_VERSION = 1;
```

Comment:
the worker check is never done. Should we create a jira for the worker client to use ClientBase()?